### PR TITLE
dmarc-metrics-exporter: 0.9.1 -> 0.9.4

### PR DESCRIPTION
**Pull request by Ma27**
## Description of changes

Changes: 
* https://github.com/jgosmann/dmarc-metrics-exporter/releases/tag/v0.9.2
* https://github.com/jgosmann/dmarc-metrics-exporter/releases/tag/v0.9.3
* https://github.com/jgosmann/dmarc-metrics-exporter/releases/tag/v0.9.4

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/xsdata/default.nix [hunk 1]
```diff
@@ -57,6 +48,8 @@ buildPythonPackage rec {
     ];
   };
 
+  propagatedBuildInputs = [ setuptools typing-extensions ];
```
Comment (dotlambda) [inline pkgs/development/python-modules/xsdata/default.nix:51]: `setuptools` shouldn't be propagated

File: pkgs/development/python-modules/xsdata/default.nix [hunk 2]
```diff
@@ -15,29 +16,19 @@
 
 buildPythonPackage rec {
   pname = "xsdata";
-  version = "22.12";
+  version = "23.8";
 
   disabled = pythonOlder "3.7";
```
Comment (dotlambda) [inline pkgs/development/python-modules/xsdata/default.nix:21]:
```suggestion
  disabled = pythonOlder "3.8";
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/xsdata/default.nix [hunk 3]
```diff
@@ -48,7 +48,8 @@ buildPythonPackage rec {
     ];
   };
 
-  propagatedBuildInputs = [ setuptools typing-extensions ];
+  buildInputs = [ setuptools ];
```
Comment (dotlambda) [inline pkgs/development/python-modules/xsdata/default.nix:51]:
```suggestion
  nativeBuildInputs = [ setuptools ];

```

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/xsdata/default.nix (hunk 3 above)
Comment (Ma27) [reply pkgs/development/python-modules/xsdata/default.nix]: Ouch, you're absolutely right, sorry!

Comment (dotlambda) [reply pkgs/development/python-modules/xsdata/default.nix]: native *build* inputs
Also, please squash.

Comment (Ma27) [reply pkgs/development/python-modules/xsdata/default.nix]: Heh, guess I was more tired than I thought I was, sorry. Fixed now and squashed %)

**Comment by Ma27** [comment]
cc @dotlambda anything else? Otherwise, would you mind approving this? :)

**Review by dotlambda — APPROVED** [review]

**Pull request merged**